### PR TITLE
docs: clarify package name rules

### PR DESCRIPTION
**Pull request by kenshanta**
- Add missing information as per Issue #1295 
- use secure hyperText Transfer Protocol : https

### CODE REVIEW

**Comment by kamil198931** [comment]
165YWbAmJG6HMCLkdTKr3DRtTc7d9CXqmU

pon., 14 paź 2024, 13:00 użytkownik Sarkis Kovlekjian <
***@***.***> napisał:

File: content/packages-and-modules/contributing-packages-to-the-registry/creating-a-package-json-file.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ A `package.json` file:
 
 A `package.json` file must contain `"name"` and `"version"` fields.
 
-The `"name"` field contains your package's name, and must be lowercase and one word, and may contain hyphens and underscores.
+The `"name"` field contains your package's name, and must be lowercase and one word, and may contain hyphens, dots and underscores.
```
Comment (wraithgar) [inline content/packages-and-modules/contributing-packages-to-the-registry/creating-a-package-json-file.mdx:28]:
```suggestion
The `"name"` field contains your package's name, and must be lowercase, not contain any spaces, and may contain hyphens, dots, and underscores.
```

"one word" seemed confusing, cause `my-cool-package` is definitely a valid package name.  What we mean is no spaces.

Also, the Oxford comma is very near and dear to me so let's include it.

**Review by wraithgar — APPROVED** [review]

**Comment by wraithgar** [comment]
Thanks! I know copy editing isn't everyone's favorite thing to do, this is much appreciated.

**Comment by kenshanta** [comment]
> Thanks! I know copy editing isn't everyone's favorite thing to do, this is much appreciated.

Happy to give back to the community!

**Pull request merged**